src/network/clientpackethandler.cpp

```diff
@@ -48,6 +48,22 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "particles.h"
 #include <memory>
 
+const char *accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
+	gettext("Invalid password"),
```

Comment:
IMO it would be better to use `N_` here and use `gettext(...)` for `m_access_denied_reason` like you previously did. This makes things easier for #14379 (if anyone adopts that).

A similar pattern can be found in e.g.
https://github.com/minetest/minetest/blob/d733e1abea1ff18c0e6375e6f05b0c29b679a3fc/src/client/keycode.cpp#L99-L104
https://github.com/minetest/minetest/blob/d733e1abea1ff18c0e6375e6f05b0c29b679a3fc/src/gui/guiKeyChangeMenu.cpp#L259

Reply:
Also, I'm not sure whether gettext is already set up with the right locale at the point where this runs.

Reply:
Applied in 59a0d95b0